common/credentials/src/coconut/bandwidth/issued.rs

```diff
@@ -137,6 +117,27 @@ impl IssuedBandwidthCredential {
         CURRENT_SERIALIZATION_REVISION
     }
 
+    pub fn expiration_date(&self) -> u64 {
+        self.expiration_date
+    }
+
+    pub fn expiration_date_formatted(&self) -> OffsetDateTime {
+        //SAFETY : expiration date is encoded as a u64 but it is a unix timestamp. The unwrap is guaranteed to succeed for at least 290 million more years
+        OffsetDateTime::from_unix_timestamp(self.expiration_date.try_into().unwrap()).unwrap()
+    }
+
+    pub fn expired(&self) -> bool {
+        self.expiration_date < today_timestamp()
+    }
+
+    pub fn exp_date_sigs(&self) -> Vec<ExpirationDateSignature> {
```

Comment:
do we have to clone it?

Reply:
We don't. It affect spending, so let's keep that for later